vendor/engines/saas/app/jobs/saas/trials_create_job.rb

```diff
@@ -0,0 +1,10 @@
+module Saas
+  class TrialsCreateJob < ActiveJob::Base
+
+    def perform(params)
+      user = params['current_user']['login']
+      params['url'] = "/settings/#{user}/trialed"
+      Analytics::PageJob.perform_later(params)
+    end
```

Comment:
Is it correct to be using `login` here if a trial is being initiated for an organization?